spec/android_localize_helper_spec.rb

```diff
@@ -8,14 +8,31 @@
     end
 
     it 'returns nil when the key is not present' do
+      test_file_content = 'my-test-key-foo = \'foo\''
+      test_file_content += 'my-test-key-bad = \'bad\''
+      test_file_content += 'my-test-key = \'my_test_value\''
+
       allow(File).to receive(:exists?).and_return(true)
-      allow(File).to receive(:open).with('./build.gradle', 'r').and_yield(StringIO.new('my-test-key = \'my_test_value\''))
+      allow(File).to receive(:open).with('./build.gradle', 'r').and_yield(StringIO.new(test_file_content))
       expect(subject.get_library_version_from_gradle_config(import_key: 'my-test-key2')).to be_nil
     end
 
     it 'returns the key content when the key is present' do
+      test_file_content = 'my-test-key-foo = \'foo\''
+      test_file_content += 'my-test-key-bad = \'bad\''
+      test_file_content += 'my-test-key = \'my_test_value\''
```

Comment:
tip: you could use either `%{}` (to avoid having to escape each single quote) or even heredoc (`<<~EOF` … `EOF`) to make this look nicer and more readable

```suggestion
      test_file_content = <<~CONTENT
        my-test-key-foo = 'foo'
        my-test-key-bad = 'bad'
        my-test-key = 'my_test_value'
      CONTENT
```

Reply:
Ah! Nice 👍